docs/source/getting_started/getting_started.md

```diff
@@ -423,7 +379,285 @@ curl -X POST "$URL/users/u1234/personalized_documents" \
+  ```
+
+## File Upload
+
+The file upload features is _not_ enabled by default contact Xayn if you do need it.
+
+Documents are often in different formats then plain text. When ingesting you can decide to provide a
+document "file" instead of a snippet. If you do so we will try to extract snippets from the given
+file.
+
+File upload implies the [`split`](#automatic-document-splitting) option which means it might always produce documents which have multiple snippets.
+The [documents with multiple snippets chapter](#documents-with-multiple-snippets) contains more information about what this means.
+
+Follwing file formats are supported: `application/pdf`, `text/plain`, `text/html`.
+
+The "file" needs to be provided as base64 encoded string.
+Reusing the example from the getting started section and provding the snippet as `text/plain` file leads to following example:
+
+```bash
+curl -X POST "$URL/documents" \
+    --header "authorizationToken: $BACKOFFICE_TOKEN" \
+    --header "Content-Type: application/json" \
+    --data '{
+        "documents": [
+            {
+                "id": "xayn_cd5604c",
+                "file": "VGhlIHZvaWNlcyB0aGF0IGFyZSBkZW1hbmRpbmcgYmV0dGVyIHByaXZhY3kgcHJvdGVjdGlvbiBhbmQgb3duZXJzaGlwIG9mIG91ciBvd24gZGF0YSBhcmUgaW5jcmVhc2luZ2x5IGxvdWRlciwgdGhlcmUgaXMgYSBiYWNrbGFzaCB0b3dhcmRzIHRoZXNlIHByYWN0aWNlcy4gQXQgWGF5biwgb3VyIG1pc3Npb24gaXMgdG8gcHJvdmlkZSBwZXJzb25hbGlzYXRpb24gd2l0aG91dCB1c2VyIGRhdGEgbGVhdmluZyB0aGUgZGV2aWNlLCBtYWludGFpbmluZyBhYnNvbHV0ZSBwcml2YWN5LiBXZSB1c2Ugc2VtYW50aWMgc2ltaWxhcml0eSBhbmQgY2VudGVycyBvZiBpbnRlcmVzdCB0byB1bmRlcnN0YW5kIHVzZXIgcHJlZmVyZW5jZXMgYW5kIHByZXNlbnQgYmV0dGVyIG1hdGNoaW5nIGFydGljbGVzLiBXaXRoIG91ciBtb2RlbCBYYXluaWEsIHdlIG9mZmVyIHNlbWFudGljIHNpbWlsYXJpdHkgYW5kIHNlYXJjaCB3aXRoIG1pbmltYWwgZW5lcmd5IGNvbnN1bXB0aW9uIGFuZCBhdCBhIGxvdyBwcmljZSwgbWFraW5nIGl0IGhpZ2hseSBlbmVyZ3ktZWZmaWNpZW50IGNvbXBhcmVkIHRvIG90aGVyIHRyYW5zZm9ybWVyIG1vZGVscy4=",
+                "properties": {
+                    "title": "The initial challange",
+                    "link": "https://xayn.com/blog/the-initial-challenge",
+                    "image": "https://uploads-ssl.webflow.com/5ef08ebd35ddb63551189655/641320bc6be72c5453f4d98d_Blog%20Posts%20Visuals%20-%2003%20Mar%202023-p-2600.png",
+                    "location" : ["germany", "berlin", "conference"]
+                }
+            }
+        ]
+    }'
+
+```
+## Summarizer
+
+The back office ingestion API provides functionality to automatically summarize the provided content.
+
+This can be used if the content of a document is too large to be used directly as a snippet. Due to the fact that
+the summarized text still has to fit into the size constraints of a snippet it is better to
+use the [automatic document splitting instead](#automatic-document-splitting).
```

Comment:
```suggestion
This can be used if the content of a document is too large to be used directly as a snippet. Because
the summarized text still has to fit into the size constraints of a snippet; it is better to
use the [automatic document splitting instead](#automatic-document-splitting).
```